Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -119,22 +122,36 @@ public bool TryGetXComponentCorrelationId(string instrumentationKey, out string
                             {
                                 this.GenerateCorrelationIdAndAddToDictionary(instrumentationKey, appId.Result);
                             }
+#if !NETCORE
                             catch (Exception ex)
                             {
                                 CrossComponentCorrelationEventSource.Log.FetchAppIdFailed(this.GetExceptionDetailString(ex));
                             }
+#else
+                            catch (Exception)
```

Comment:
(I agree completely with what you're saying. I just don't know what other options are available.)